src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/common/CommonSteps.scala

```diff
@@ -36,6 +39,23 @@ object CommonSteps {
       }
       .pause(30 second)
 
+  def provisionUsersWithMailboxesAndMessages(users: Seq[Future[User]]) =
+    scenario("ProvisionUsersWithMailboxesAndMessages")
+      .exec(provisionSystemMailboxes(users))
+      .repeat(NumberOfMailboxes) {
+        var mailboxName = Name()
+        exec(JmapMailboxes.createMailbox(Id(), mailboxName))
+          .pause(1 second, 2 seconds)
```

Comment:
What does that mean you pause for 1 second, 2 seconds or 3 seconds. Or something random between 1 and 2 seconds ?

Reply:
Random between 1 and 2